test/validate_test.js

```diff
@@ -689,7 +689,7 @@ describe("Validation", () => {
           validate,
           liveValidate: true,
         });
-        comp.componentWillReceiveProps({ formData });
+        comp.componentWillReceiveProps({ ...comp.props, formData });
 
         expect(comp.state.errorSchema).eql({
           0: { __errors: [] },
```

Comment:
Can you add an actual test, if possible, that actually verifies this behavior?
- if the parent component changes, and none of the props change, the form should not rerender
- if the parent component changes, which causes one of the props to change, the form should rerender

Reply:
I am not familiar with the test renderer you have set up.  I am not sure how to ask it to tell us if a rerender happened or not.  Possibly we can use a spy on the render function, or we could also test by passing in a validate function and ensuring it is not called if the data doesn't change.